fern/definition/domaintakeover.yml

```diff
@@ -0,0 +1,30 @@
+types:
+  Fingerprint:
+    properties:
+      cicdPass: boolean
+      cname: list<string>
+      discussion: string
+      documentation: string
+      fingerprint: string
+      httpStatus: optional<integer>
+      nxDomain: boolean
+      service: string
+      status: string
+      vulnerable: boolean
+  Service:
+    properties:
+      name: string
+      fingerprint: string
+      vulnerable: boolean
+  DomainTakeover:
+    properties:
+      target: string
+      statusCode: integer
+      responseBody: string
+      domain: string
+      cname: string
+      services: list<Service>
```

Comment:
Does this need to be optional?

Reply:
No if there is no error, the object will always generate this list of services and whether or not they match a Domain Takeover fingerprint. If there is an error an empty report is sent back